Eatery/Controllers/BRB/BRBAccountViewController.swift

```diff
@@ -8,16 +8,22 @@
 
 import UIKit
 
+protocol BRBAccountViewControllerDelegate {
+    func brbAccountViewControllerDidRefresh()
+}
+
 class BRBAccountViewController: UIViewController {
     
     private enum CellIdentifiers {
         static let balance = "balance"
         static let history = "history"
     }
     
-    private let account: BRBAccount
+    var account: BRBAccount
+    var delegate: BRBAccountViewControllerDelegate?
     
-    private var tableView: UITableView!
+    var tableView: UITableView!
+    var refreshControl: UIRefreshControl!
```

Comment:
Can we get rid of this `refreshControl` variable since we assign it to `tableView.refreshControl`?